### PR TITLE
html, css title, question card and nav bar

### PR DESCRIPTION
**Pull request by henkegithub**
I've added the html code and styled the following via css:

title, question card and nav bar.

can anyone help me with the perfect styling of the nav bar and how to make the icons clickable?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,27 +15,33 @@
...
         <!--Question Card
         with questions, bookmarks, 
         categories as tags, 
         answer shows on a click
         -->
-        <p>What property flips the axes in flexbox?</p>
-        <button>Show Answer</button>
-        #html #flexbox #css
+        <p class="question">What property flips the axes in flexbox?</p>
+        <input class="answer" type="checkbox">Show Answer</input>
```
Comment (F-Kirchhoff) [inline index.html]: ahh sweet ;)

Comment (F-Kirchhoff) [reply index.html]: although the class is missleading, it should be something like "answer-button"

Comment (F-Kirchhoff) [reply index.html]: And normally you would put the text into the label of the input

File: index.html [hunk 2]
```diff
@@ -15,27 +15,33 @@
...
       <nav class="navigationbar">
-        <ul>
-          <li><img src="assets/homebutton.png" /></li>
-          <li><img src="assets/safe button.png" /></li>
-          <li><img src="assets/profile.png" /></li>
+        <ul class="images">
+          <li class="imagebox"><img src="assets/homebutton.png" /></li>
+          <li class="imagebox"><img src="assets/safe button.png" /></li>
+          <li class="imagebox"><img src="assets/profile.png" /></li>
         </ul>
```
Comment (F-Kirchhoff) [inline index.html]: your navigation should include anchor tags 'a' since they will redirect you to another page.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
This is already very good stuff. Since you are using the checkbox hack, it is necessarily not accessible but good for now. Try to put the text of the "answer button" into the label of the checkbox and make the input a self closing tag.

File: index.html (hunk 1 above)
Comment (henkegithub) [reply index.html:25]: so basically something like this?
<input class="answer-button" type="checkbox" checkbox="Show Answer"/>

File: index.html (hunk 2 above)
Comment (henkegithub) [reply index.html]: something like this?
`<li class="imagebox"><a href=""></a></li>`

**Review by henkegithub — COMMENTED** [review]
so I added the checkbox hack. 

how can I make It so after I click the view answer tag changes to hide answer and back?

is the anchor tag connected with the list and images correct?

File: css/styles.css [hunk 3]
```diff
@@ -58,4 +58,22 @@
...
+
+label {
+  display: flex;
+  justify-content: center;
+}
+
+.visually-hidden {
+  position: absolute;
+  left: -100vw;
+}
```
Comment (F-Kirchhoff) [inline css/styles.css]: This is bad practice, you can use display: none or visibility: hidden

File: css/styles.css [hunk 4]
```diff
@@ -58,4 +58,22 @@
   border: solid;
 }
 
+.control-me::after {
+  content: "";
+  font-size: 100px;
+}
```
Comment (F-Kirchhoff) [inline css/styles.css]: I think you don't need this ruleset

Comment (F-Kirchhoff) [reply css/styles.css]: You can instead use the display: none property to hide the answer and set the display to "block" when the checkmark is set.

File: index.html [hunk 5]
```diff
@@ -22,26 +22,44 @@ <h1>Quizz App</h1>
...
+          <li class="hashtagbox">#html</li>
+          <li class="hashtagbox">#flexbox</li>
+          <li class="hashtagbox">#css</li>
         </ul>
       </section>
       <section class="card">
         <p class="question">Lorem ipsum dolor sit?</p>
-        <input class="answer" type="checkbox">Show Answer</input>
-        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora aspernatur autem, necessitatibus dolore quae fuga! Labore quod expedita, dolore deleniti nesciunt quas, eum commodi perferendis ipsa enim non totam mollitia?</p>
+        <p>show answer</p>
```
Comment (F-Kirchhoff) [inline index.html]: This should be a button, since it is expected to be pressed.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
A bit better, but there were still some issues. Please have a look!

**Pull request merged**